projects/frontend/data-pipelines/gui/projects/data-pipelines/src/lib/shared/utils/data-job.util.ts

```diff
@@ -21,9 +21,9 @@ export class DataJobUtil {
     /**
      * ** Predicate for Job Running.
      */
-    static isJobRunningPredicate(
+    static isJobRunningPredicate = (
```

Comment:
Why changing it from method to class field function?
Intercepting and spying on methods in unit tests is much more easier.

If it is declared with getter and setter would be easier to spy in unit tests, but field declared like your change I think is not possible to spy on.

Reply:
It's the suggested fix for this linter warning

https://typescript-eslint.io/rules/unbound-method/

Also, spying on them seems to work fine, at least when I run tests locally. The unit test I've linked below doesn't break.

https://github.com/vmware/versatile-data-kit/blob/main/projects/frontend/data-pipelines/gui/projects/data-pipelines/src/lib/shared/utils/data-job.util.spec.ts#L80

Reply:
My personal opinion for static methods that are fail safe because not using this reference, is to not not be transformed to class fields with bound arrow methods.